cmds/fusermount/fusermount_linux.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"os"
+
+	"golang.org/x/sys/unix"
+)
+
+var (
+	fileSystemUID, fileSystemGID int
```

Comment:
Rather than using globals, have `type FSPrivs struct {uid, gid int}` and change the signatures to `func dropPrivs() (old FSPrivs, err error)` and `func restorePrivs(old FSPrivs)`.